src/main/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCaller.scala

```diff
@@ -171,8 +172,8 @@ object GermlineAssemblyCaller {
 
       // Build a variant using the current offset and read evidence
       def buildVariant(referenceOffset: Int,
-                       referenceBases: Array[Byte],
```

Comment:
Also a recurring pattern, so I won't mention it more, but unless the sequence is actually indexed by genomic locus I think it adds confusion to call it a `ContigSequence`